_targets.R

```diff
@@ -47,7 +43,12 @@ tar_target(
   nwis_01466500_data_csv,
   download_nwis_site_data(filepath = '1_fetch/out/nwis_01466500_data.csv'),
   format = "file"
-  ),                                                # site_data is the concatenatd dataframe of the seperate csv files, should get this to return
+  ),
+tar_target(
+  site_data,
+  download_nwis_data(site_nums = site_nums),       # where this function is now focused on getting the site_data, I think this function needs to be called first
```

Comment:
I see you've changed the function, but maybe change the function name to reflect what it's doing now? e.g., binding data, not downloading?